packages/manager/modules/pci/src/projects/project/users/users.service.js

```diff
@@ -14,6 +16,11 @@ export default class PciProjectsProjectUsersService {
     this.OvhApiCloudProject = OvhApiCloudProject;
     this.OvhApiCloudProjectRegion = OvhApiCloudProjectRegion;
     this.OvhApiCloudProjectUser = OvhApiCloudProjectUser;
+    this.ALPHA_CHARACTERS_REGEX = ALPHA_CHARACTERS_REGEX;
+  }
+
+  checkGlobalRegion(regions) {
```

Comment:
What does this do ?

Reply:
We have global regions now which are GRA, DE, etc... This checks if global regions are present or not in regions list and returns boolean